prime-factors/prime_factors.py

```diff
@@ -1,2 +1,10 @@
 def prime_factors(natural_number):
-    pass
+    list = []
+    x = 2
+    while x <= natural_number:
+        if natural_number % x == 0:
+            list.append(x)
+            natural_number = natural_number/x
```

Comment:
@zarniak Same trick as with shortening `x = x + 1` to `x += 1` works here as well 💡But that's just syntactic sugar and the code works just fine without it.